rs/ethereum/cketh/minter/templates/pagination.html

```diff
@@ -0,0 +1,12 @@
+<tr>
+    <td colspan='{{ table_width }}' style='text-align: right;'>
+        <b>Pages: </b>
+        {% for page in pages %}
+        {% if page.index == current_page_index %}
+        {{ page.index }}&nbsp;
+        {% else %}
+        <a href='?{{ page_offset_query_param }}={{ page.offset }}#{{ table_id }}'>{{ page.index }}</a>&nbsp;
```

Comment:
This seems to discard the pagination data for other tables. So is it not possible to view both page 3 of one table and page 2 of another table?

Reply:
@ninegua Good question! You are correct. The problem is, afaik it is not trivial to add a query parameter without discarding the existing ones. It might be possible with a little bit of JS magic (see e.g. [here](https://stackoverflow.com/questions/6899097/how-to-add-a-parameter-to-the-url)), but I guess the question is whether it is worth the time. What do you think?